src/test/max_matching_test.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved
```

Comment:
Unless you're just renaming an existing test, this should be 2020, The Souffle Developers

Reply:
Fixed. Great catch.